.eslintrc.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
   env: {
     browser: true,
     es6: true,
+    "jest": true,
```

Comment:
Why is this required? Linting is still performing accurately without setting jest in the env!